Formula/streamsets.rb

```diff
@@ -0,0 +1,50 @@
+class Streamsets < Formula
+  desc "ETL tool with a graphical IDE for building dataflow pipelines"
+  homepage "https://streamsets.com/"
+  url "https://archives.streamsets.com/datacollector/1.5.1.0/tarball/streamsets-datacollector-all-1.5.1.0.tgz"
+  sha256 "d848ffaec4cbadce0e7aa031fb30d1e7e13c84be9115ecd6c69643ff02400fc8"
+
+  bottle :unneeded
+
+  depends_on :java => "1.7+"
+  depends_on "md5sha1sum"
+
+  def install
+    # rename old SDC_CONF if it exists
+    if File.exist?("#{var}/lib/sdc")
```

Comment:
We normally do this sort of thing in `post_install`. I guess it's acceptable here because there's no bottle process, but unsure if it's worth moving to `post_install` anyway in case there is a bottle requirement at a later date.